res/configuration.toml

```diff
@@ -1,68 +1,104 @@
+# so Database credentials can be pulled from Vault.
+# Note when running in docker from compose file set the following environment variables:
+#   - SECRETSTORE_HOST: edgex-vault
+[SecretStore]
+Type = "vault"
+Host = "localhost"
+Port = 8200
+Path = "app-rfid-llrp-inventory/"
+Protocol = "http"
+TokenFile = "/tmp/edgex/secrets/app-rfid-llrp-inventory/secrets-token.json"
+RootCaCertPath = ""
+ServerName = ""
+  [SecretStore.Authentication]
+  AuthType = "X-Vault-Token"
 
-[Binding]
-Type="messagebus"
-SubscribeTopic="events"
-PublishTopic="events"
+[Trigger]
+Type="edgex-messagebus"
+  [Trigger.EdgexMessageBus]
+  Type = "redis"
+    [Trigger.EdgexMessageBus.SubscribeHost]
+    Host = "localhost"
+    Port = 6379
+    Protocol = "redis"
+    SubscribeTopics="edgex/events/#/#/#/ROAccessReport,edgex/events/#/#/#/ReaderEventNotification"
+    [Trigger.EdgexMessageBus.Optional]
+    authmode = "usernamepassword"  # required for redis messagebus (secure or insecure).
+    secretname = "redisdb"
+    # Default MQTT Specific options that need to be here to enable evnironment variable overrides of them
+    # Client Identifiers
+    ClientId ="app-rules-engine"
+    # Connection information
+    Qos          =  "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+    KeepAlive    =  "10" # Seconds (must be 2 or greater)
+    Retained     = "false"
+    AutoReconnect  = "true"
+    ConnectTimeout = "5" # Seconds
+    # TLS configuration - Only used if Cert/Key file or Cert/Key PEMblock are specified
+    SkipCertVerify = "false"
 
-# Every device(reader) + antenna port represents a tag location and can be assigned an alias
-# such as Freezer, Backroom etc. to give more meaning to the data. The default alias set by
-# the application has a format of <deviceName>_<antennaId> e.g. Reader-10-EF-25_1 where
-# Reader-10-EF-25 is the deviceName and 1 is the antennaId.
-# See also: https://github.com/edgexfoundry/app-rfid-llrp-inventory#setting-the-aliases
-#
-# In order to override an alias, set the default alias as the key, and the new alias as the value you want, such as:
-# Reader-10-EF-25_1 = "Freezer"
-# Reader-10-EF-25_2 = "Backroom"
-[Aliases]
+[AppCustom]
+  # Every device(reader) + antenna port represents a tag location and can be assigned an alias
+  # such as Freezer, Backroom etc. to give more meaning to the data. The default alias set by
+  # the application has a format of <deviceName>_<antennaId> e.g. Reader-10-EF-25_1 where
+  # Reader-10-EF-25 is the deviceName and 1 is the antennaId.
+  # See also: https://github.com/edgexfoundry/app-rfid-llrp-inventory#setting-the-aliases
+  #
+  # In order to override an alias, set the default alias as the key, and the new alias as the value you want, such as:
+  # Reader-10-EF-25_1 = "Freezer"
+  # Reader-10-EF-25_2 = "Backroom"
+  [AppCustom.Aliases]
+  Reader-10-EF-25_1 = "Freezer"
+  Reader-10-EF-25_2 = "Backroom"
+  Reader-3 = "my truck"
```

Comment:
As cool as your truck probably is :smile: , this is not a proper naming scheme:

> The default alias set by the application has a format of `<deviceName>_<antennaId>` e.g. `Reader-10-EF-25_1` where `Reader-10-EF-25` is the `deviceName` and `1` is the `antennaId`.
> ...
> In order to override an alias, _set the default alias as the key_, and the new alias as the value you want

Reply:
oops forgot to remove after testing